lib/auth/login_log.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Auth.LoginLog do
+  use Ecto.Schema
+  import Ecto.Changeset
+  alias Auth.Repo
+
+  schema "login_logs" do
+    field :email, :string
+    field :ip_address, :string
```

Comment:
IP Address is considered [PII](https://en.wikipedia.org/wiki/Personal_data) for GDPR and in this case we want to store it _encrypted_ (_so that we can decrypt it in case of needing to inspect for malicious activity_).
Please consider doing: https://github.com/dwyl/fields/issues/32

Reply:
I've now updated the field to use the new `IpAddressEncrypted` field type